src/main/java/bio/terra/service/dataset/flight/ingest/IngestUtils.java

```diff
@@ -205,6 +205,10 @@ public static PdaoLoadStatistics getIngestStatistics(FlightContext context) {
         return false;
       };
 
+  public static final Predicate<FlightContext> noCopyNeeded =
+      flightContext ->
+          !flightContext.getWorkingMap().get(IngestMapKeys.CONTROL_FILE_NEEDS_COPY, Boolean.class);
```

Comment:
Same as before, I would write this as a function, not as a `Predicate`
```suggestion
  public static boolean noCopyNeeded(FlightContext flightContext) {
    return !flightContext.getWorkingMap().get(IngestMapKeys.CONTROL_FILE_NEEDS_COPY, Boolean.class);
  }
```